xknx/devices/climate.py

```diff
@@ -58,13 +57,6 @@ def __init__(
         """Initialize Climate class."""
         # pylint: disable=too-many-arguments, too-many-locals, too-many-branches, too-many-statements
         super().__init__(xknx, name, device_updated_cb)
-        if isinstance(group_address_on_off, (str, int)):
```

Comment:
Why is this removed?

Reply:
now we can pass GroupAddress() objects to all group_address_* attributes. If something unparsable is passed, an error will be raised when instantiating the GroupAddress in RemoteValue.
There should be no need to double-check types here.

Reply:
Ah, now it makes sense. We only use that in the constructor.